src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
+  margin: 0 auto;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  --secondColor: #00acdc;
```

Comment:
A common best practice is to define custom properties on the [:root] pseudo-class, so that it can be applied globally across your HTML document:
```
:root {
   --secondary-color: #00acdc;
}
```

also name variables using cebab case as well like --secondary-color